npm/ng-packs/packages/components/extensible/src/lib/components/extensible-form/extensible-form.component.ts

```diff
@@ -44,11 +44,13 @@ export class ExtensibleFormComponent<R = any> {
         const type = !record || JSON.stringify(record) === '{}' ? 'create' : 'edit';
         const propList = this.extensions[`${type}FormProps`].get(this.identifier).props;
         this.groupedPropList = this.createGroupedList(propList);
+        this.groupedPropListOfArray = this.groupedPropList.items.map(i => i.formPropList.toArray());
```

Comment:
Do we really need this ? we can already reach `formPropList` property in `groupedPropList` use this in `isAnyGroupMemberVisible` ?

Reply:
`formPropList` is linked list, so i need to convert this to array so i can traverse it easly

Reply:
My problem not about casting, it's about the why you define new variable for this ? You using only in `isAnyGroupMemberVisible`

Reply:
i understood what you mean, i will remove unnecesarry variable